api/base/views.py

```diff
@@ -368,357 +368,11 @@ def create(self, *args, **kwargs):
-    descriptive error in the body of the response.  The response body will be an object with a key, `errors`, pointing
-    to an array of error objects.  Generally, these error objects will consist of a `detail` key with a detailed error
-    message and a `source` object that may contain a field `pointer` that is a [JSON
-    Pointer](https://tools.ietf.org/html/rfc6901) to the error-causing attribute. The `error` objects may include
-    additional information in accordance with the [JSON-API error spec](http://jsonapi.org/format/1.0/#error-objects).
-
-    ####Example: Error response from an incorrect create node request
-
-        {
-          "errors": [
-            {
-              "source": {
-                "pointer": "/data/attributes/category"
-              },
-              "detail": "This field is required."
-            },
-            {
-              "source": {
-                "pointer": "/data/type"
-              },
-              "detail": "This field may not be null."
-            },
-            {
-              "source": {
-                "pointer": "/data/attributes/title"
-              },
-              "detail": "This field is required."
-            }
-          ]
-        }
-
-    ##OSF Enum Fields
-
-    Some entities in the OSF API have fields that only take a restricted set of values.  Those fields are listed here
-    for reference.  Fuller descriptions are available on the relevant entity pages.
-
-    ###OSF Node Categories
-
-        value                 description
-        ==========================================
-        project               Project
-        hypothesis            Hypothesis
-        methods and measures  Methods and Measures
-        procedure             Procedure
-        instrumentation       Instrumentation
-        data                  Data
-        analysis              Analysis
-        communication         Communication
-        other                 Other
-
-    ###OSF Node Permission keys
-
-        value        description
-        ==========================================
-        read         Read-only access
-        write        Write access (make changes, cannot delete)
-        admin        Admin access (full write, create, delete, contributor add)
-
-    ###Storage Providers
-
-    Valid storage providers are:
+    """
+    The documentation for this endpoint can be found [here](https://developer.osf.io/#Base_base_read).
 
-        value        description
-        ==========================================
-        box          Box.com
-        dataverse    Dataverse
-        dropbox      Dropbox
-        figshare     figshare
-        github       GitHub
-        googledrive  Google Drive
-        osfstorage   OSF Storage
-        s3           Amazon S3
 
+    The documentation for the Open Science Framework API can be found at `https://developer.osf.io`.
```

Comment:
Can you move this line above the other, so that the link to developer.osf.io comes before the link for this endpoint? 

Also, I think the link in this line should be `[developer.osf.io](https://developer.osf.io)`, so that it's rendered as a link and not a code snippet.